.github/workflows/pr-preview.yml

```diff
@@ -0,0 +1,71 @@
+name: deploy-preview
+
+on:
+  pull_request:
+    branches:
+      - main
+
+env:
+  # Create a host name for the environment: mr-preview-<PR_NUMBER>.konturio.surge.sh
+  DOMAIN: https://mr-preview-${{ github.event.number }}-konturio.surge.sh
+
+jobs:
+  deploy:
+    runs-on: ubuntu-24.04
+    env:
+      PREVIEW_API_DOMAIN: 'test-apps-ninja02.konturlabs.com'
+      KEYCLOAK_URL: 'https://dev-keycloak.k8s-01.konturlabs.com'
+      KEYCLOAK_REALM: 'dev'
+
+    steps:
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v4.0.3
+        with:
+          node-version: 20
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4.0.0
+        with:
+          version: ^9.x.x
+
+      - name: Checkout
+        uses: actions/checkout@v4.1.7
+
+      - name: Cache pnpm store
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.pnpm-store
+            node_modules
+          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-store-
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Apply domain changes in config files
+        run: |
+          sed -i "s|https://disaster.ninja|https://${PREVIEW_API_DOMAIN}|" configs/config.default.json
+          sed -i "s|\"KEYCLOAK_URL\": \"https://keycloak01.kontur.io\"|\"KEYCLOAK_URL\": \"${KEYCLOAK_URL}\"|" configs/config.default.json
+          sed -i "s|\"KEYCLOAK_REALM\": \"kontur\"|\"KEYCLOAK_REALM\": \"${KEYCLOAK_REALM}\"|" configs/config.default.json
+
+      - name: Update .env.production
+        run: |
+          sed -i "s|VITE_BASE_PATH=/active/|VITE_BASE_PATH=/|" .env.production
+          sed -i "s|VITE_STATIC_PATH=static/|VITE_STATIC_PATH=|" .env.production
+
+      - name: Build
+        run: pnpm build
+
+      - name: Enable surge.sh redirects
+        run: cp dist/index.html dist/200.html
+
+      - name: Deploy to surge.sh
+        if: ${{ !cancelled() }}
+        uses: dswistowski/surge-sh-action@v1
+        with:
+          project: ./dist
+          domain: ${{ env.DOMAIN }}
+          login: ${{secrets.SURGE_EMAIL}}
+          token: ${{secrets.SURGE_TOKEN}}
```

Comment:
@dqunbp sure the same as for autotests domain token will work ?

Reply:
Yes, it works, domain defined [here](https://github.com/konturio/disaster-ninja-fe/pull/794/files#diff-737d2247cd4b3b6578d2c5413b555a5ca90f6aaa32613770ffdc146900d0469bR10)